catalog/wcag2-catalog.yaml

```diff
@@ -0,0 +1,29 @@
+title: WCAG 2.0
+chapters:
```

Comment:
I like keeping some similarity between the catalog and content - makes things easier to parse for folks. There will be some things (like components) that we break out of the main flow, of course.